main/main.js

```diff
@@ -21,21 +45,23 @@ crashReporter.start({
 })
 
 if (process.argv.some(arg => arg === '-v' || arg === '--version')) {
-  console.log('Min: ' + app.getVersion())
-  console.log('Chromium: ' + process.versions.chrome)
+  console.log(`Min: ${app.getVersion()}`)
+  console.log(`Chromium: ${process.versions.chrome}`)
   process.exit()
 }
 
+// arg === '--development-mode' -> arg === CMD_DEV_MODE
```

Comment:
What's the purpose of this?

Reply:
Actually, constant variables again but in optional, and you can change name to anything else you want 🤗

But I'm suggesting you, using enum, I know enum isn't available in JS but we can make similar of them...

```js
const SHELL_FLAG = {
  DEVELOPER_MODE: '--developer-mode'
}
```
In usage this be like enum,
```js
SHELL_FLAG.DEVELOPER_MODE
```